packages/infra/lib/api-stack/api-service.ts

```diff
@@ -213,7 +210,6 @@ export function createAPIService({
           LB_TIMEOUT_IN_MILLIS: loadBalancerIdleTimeout.toMilliseconds().toString(),
           DB_READ_REPLICA_ENDPOINT: dbReadReplicaEndpointAsString,
           DB_POOL_SETTINGS: dbPoolSettings,
-          TOKEN_TABLE_NAME: dynamoDBTokenTable.tableName,
```

Comment:
We'll need to update the code so `getTokenTableName()` can return `undefined` and uses `getEnvVar` instead of `getEnvVarOrFail`